kratos/tests/cpp_tests/external_libraries/csv-parser/test_csv_buffer.cpp

```diff
@@ -0,0 +1,126 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Vincent La (This an adaptation of https://github.com/vincentlaucsb/csv-parser)
+//                   Vicente Mataix Ferrandiz
+//
+//
+
+#ifdef KRATOS_BUILD_CSV_TESTING
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "testing/testing.h"
+#include "csv-parser/include/csv.hpp"
```

Comment:
can you move the file so that the include path is shorter?
=> `#include "csv-parser/csv.hpp"`